docs/concepts/hybrid-approach.md

```diff
@@ -62,11 +63,11 @@ This mapping table is also used in the browser-side PWA to switch from the singl
 
 ### Intershop Commerce Management (ICM)
 
-ICM must be run with [Secure URL-only Server Configuration](https://docs.intershop.com/icm/7.10/olh/oma/en/topics/managing_site_settings/topic_secure_url-only_server_configuration.html), which can be done by adding `SecureAccessOnly=true` to the appserver configuration.
+ICM must be run with [Secure URL-only Server Configuration](https://knowledge.intershop.com/oh/index.php/Display/40633N), which can be done by adding `SecureAccessOnly=true` to the appserver configuration.
```

Comment:
Shouldn't we link to the latest document version instead of 7.10 here as well?
Would be https://knowledge.intershop.com/oh/index.php/Display/3366K3